common/datasets/tedlium2/corpus.py

```diff
@@ -0,0 +1,87 @@
+import os
+from functools import lru_cache
+from typing import Dict, Optional
+
+from sisyphus import tk
+
+from i6_core.audio.encoding import BlissChangeEncodingJob
+
+from i6_core.meta import CorpusObject
+
+from .constants import DURATIONS
+from .download import download_data_dict
+
+
+@lru_cache()
+def get_bliss_corpus_dict(
+    audio_format: str = "wav", output_prefix: str = "datasets"
+) -> Dict[str, tk.Path]:
+    """
+    :param audio_format: options: wav, ogg, flac, sph, nist. nist (NIST sphere format) and sph are the same.
+    :param output_prefix:
+    :return:
+    """
+    assert audio_format in ["flac", "ogg", "wav", "sph", "nist"]
+
+    output_prefix = os.path.join(output_prefix, "Ted-Lium-2")
+
+    bliss_corpus_dict = download_data_dict(output_prefix=output_prefix)["bliss_nist"]
+
+    audio_format_options = {
+        "wav": {
+            "output_format": "wav",
+            "codec": "pcm_s16le",
+        },
+        "ogg": {"output_format": "ogg", "codec": "libvorbis"},
+        "flac": {"output_format": "flac", "codec": "flac"},
+    }
+
+    converted_bliss_corpus_dict = {}
+    if audio_format != "sph" or audio_format != "nist":
```

Comment:
This is always `True`, isn't it? Do you mean `if audio_format not in ["sph", "nist"]`?